packages/pcd/eddsa-pcd/src/util/util.ts

```diff
@@ -33,7 +81,58 @@ export function isEdDSAPublicKey(
     !isNaN(Number("0x" + maybeKey[1]))
   ) {
     return true;
+  } else if (typeof maybeKey === "string") {
+    try {
+      checkPublicKeyFormat(maybeKey);
+      return true;
+    } catch (_e) {
+      return false;
+    }
   } else {
     return false;
   }
 }
+
```

Comment:
Agreed with your suggestion in the PR description that much of this code should migrate into a library which can be easily shared between EdDSAPCD and POD.  My podCrypto file was a nod toward that, but I didn't have a need to turn it into a library yet.

For a short-term fix to the circularity, though, the backward-compatibility tests could move out of the POD library into this library.  I think that's the only reason POD code should depend on this PCD.